### PR TITLE
Issue #9551: update example of AST for TokenTypes.PARAMETERS

### LINKED ISSUE #9551: update example of AST for TokenTypes.PARAMETERS
https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#PARAMETERS

needs to be extended to show example of AST.
Source to be updated: https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

PLEASE READ for more details: https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-AST-in-TokenTypes

### PR DESCRIPTION
**Pull request by Whitroy**
Closes #9551 

Capture1

Code used for generating example:
```
public class ParameterTest{
    public void add(int num1, int num2){}
}
```
Output:
```
java -jar checkstyle-8.41.1-all.jar -t ParameterTest.java

CLASS_DEF -> CLASS_DEF [1:0]
|--MODIFIERS -> MODIFIERS [1:0]
|   `--LITERAL_PUBLIC -> public [1:0]
|--LITERAL_CLASS -> class [1:7]
|--IDENT -> ParameterTest [1:13]
`--OBJBLOCK -> OBJBLOCK [1:26]
    |--LCURLY -> { [1:26]
    |--METHOD_DEF -> METHOD_DEF [2:4]
    |   |--MODIFIERS -> MODIFIERS [2:4]
    |   |   `--LITERAL_PUBLIC -> public [2:4]
    |   |--TYPE -> TYPE [2:11]
    |   |   `--LITERAL_VOID -> void [2:11]
    |   |--IDENT -> add [2:16]
    |   |--LPAREN -> ( [2:19]
    |   |--PARAMETERS -> PARAMETERS [2:20]
    |   |   |--PARAMETER_DEF -> PARAMETER_DEF [2:20]
    |   |   |   |--MODIFIERS -> MODIFIERS [2:20]
    |   |   |   |--TYPE -> TYPE [2:20]
    |   |   |   |   `--LITERAL_INT -> int [2:20]
    |   |   |   `--IDENT -> num1 [2:24]
    |   |   |--COMMA -> , [2:28]
    |   |   `--PARAMETER_DEF -> PARAMETER_DEF [2:30]
    |   |       |--MODIFIERS -> MODIFIERS [2:30]
    |   |       |--TYPE -> TYPE [2:30]
    |   |       |   `--LITERAL_INT -> int [2:30]
    |   |       `--IDENT -> num2 [2:34]
    |   |--RPAREN -> ) [2:38]
    |   `--SLIST -> { [2:39]
    |       `--RCURLY -> } [3:4]
    `--RCURLY -> } [4:0]

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -628,6 +628,35 @@
...
+     *      |   |   `--IDENT -> num1
+     *      |   |--COMMA -> ,
+     *      |   `--PARAMETER_DEF -> PARAMETER_DEF
+     *      |       |--MODIFIERS -> MODIFIERS
+     *      |       |--TYPE -> TYPE
+     *      |       |   `--LITERAL_INT -> int
+     *      |       `--IDENT -> num2
+     *      |--RPAREN -> )
+     *
+     * </pre>
```
Comment (CryptoSingh1337) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:659]: @Whitroy please leave a blank line after `</pre>`

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 2]
```diff
@@ -628,6 +628,35 @@
      * be variable length (indicated by the ELLIPSIS child node immediately
      * after the TYPE child).
      *
+     * <p>For example</p>
+     * <pre>
+     * public int add(int num1, int num2)
```
Comment (CryptoSingh1337) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:633]: add `{}` after the method declaration

**Comment by CryptoSingh1337** [comment]
@Whitroy please read https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-AST-in-TokenTypes

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java (hunk 2 above)
Comment (Whitroy) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:633]: Do I have to write a complete function there? like { return num1 + num2;} as well?

Comment (CryptoSingh1337) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:633]: You can make the function return type as void and just add empty method body.

**Pull request closed without merging**